src/fides/api/task/graph_task.py

```diff
@@ -107,9 +108,20 @@ def result(*args: Any, **kwargs: Any) -> Any:
                         method_name,
                         self.execution_node.address,
                     )
-                    self.log_paused(action_type, ex)
+                    self.log_awaiting_processing(action_type, ex)
```

Comment:
There is nothing actually raising PrivacyRequestPaused now (as part of the traversal - prepolicy webhooks do raise this before the traversal).  So this code block shouldn't be hit and should be removed.